test/Makefile

```diff
@@ -0,0 +1,78 @@
+comma := ,
```

Comment:
convention elsewhere is to call the test dir `tests` by the looks of it

Reply:
Will rename